Add release notes for Java agent 7.2.0

twcrone opened the pull request:
Please merge ASAP.  Thanks!

<!-- For Japanese readers: 
もしドキュメントの日本語訳で問題を見つけた場合はPRではなくissueを提出してください。
日本語訳へのPRについてはまだ取り込む準備ができていません。-->

### Give us some context

7.2.0 Release Documentation

rhetoric101 commented on src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0:
```diff
@@ -0,0 +1,52 @@
+
+### New features and improvements
+
+* Scala Library Instrumentation [#362](https://github.com/newrelic/newrelic-java-agent/pull/362) [#363](https://github.com/newrelic/newrelic-java-agent/pull/363)
```
@twcrone, consider inserting an "and" between these two links.

rhetoric101 commented on src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0:
```diff
@@ -0,0 +1,52 @@
+  * Play 2.3-2.8
+  * Akka-HTTP v10.1 & v10.2
+  * For more information, see [Scala instrumentation](https://docs.newrelic.com/docs/agents/java-agent/frameworks/scala-installation-java/).
+
```
@twcrone, I don't think you need more than one line feed separating your lists.

rhetoric101 commented on src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0:
```diff
@@ -0,0 +1,52 @@
+  * Refactored code to use less memory.
+
+### Fixes
+The existing mongo instrumentation was partially applying when mongo reactive is being used.
```
@twcrone, consider capitalizing  and using the entire names: "MongoDB" and "MongoDB Reactive Streams."

rhetoric101 reviewed (commented):
@twcrone, thanks for adding these release notes! I just made a few minor suggestions. 

I had trouble building the site locally on my box with your changes, so I'm going to check with the DevEn team. I'll keep you posted.

rhetoric101 commented:
@twcrone, I added a `.mdx` extension to get this to build locally. 

I moved this over to drafts; so, when you are ready on the 9th, please ping the documentation hero and ask them to do another quick review and publish this.

twcrone commented:
> @twcrone, I added a `.mdx` extension to get this to build locally.
> 
> I moved this over to drafts; so, when you are ready on the 9th, please ping the documentation hero and ask them to do another quick review and publish this.

So the team has decided to move forward, so as soon as this is ready, let's merge it.  The production artifacts are already out there and available now.

twcrone commented:
Can we remove 'draft' status, review changes and get this merged please?
